cmd/tetragon/flags.go

```diff
@@ -129,4 +129,14 @@ func readAndSetFlags() {
 	option.Config.ReleasePinned = viper.GetBool(keyReleasePinnedBPF)
 	option.Config.EnablePolicyFilter = viper.GetBool(keyEnablePolicyFilter)
 	option.Config.EnablePolicyFilterDebug = viper.GetBool(keyEnablePolicyFilterDebug)
+
+	// manually handle the deprecation of --config-file
```

Comment:
Should we add a date and a version here so that we know when to fully remove it?

Reply:
Yep maybe we can say (as we discussed offline):
- deprecation for next minor release, maybe `0.10.0`
- removal for the next minor release after that, maybe `0.11.0`

Reply:
@kkourt I think it would be reasonable to have a list of things that we are planning to deprecate/remove for version N so that we can check when we make the release that we are on track. Would creating an umbrella issue for deprecation make sense?